tests/conftest.py

```diff
@@ -74,58 +74,74 @@ def base_route(api_version, gordo_project, gordo_name):
     return f"/gordo/{api_version}/{gordo_project}/{gordo_name}"
 
 
+@pytest.fixture(scope="session")
+def model_collection_directory(gordo_project: str):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        collection_dir = os.path.join(tmp_dir, gordo_project)
+        os.makedirs(collection_dir, exist_ok=True)
+        yield collection_dir
+
+
+@pytest.fixture(scope="session")
+def config_str(gordo_name: str, sensors: List[SensorTag]):
+    """
+    Fixture: Default config for testing
+    """
+    yield f"""
```

Comment:
Is there a functional difference between using `yield` vs `return` here?

Reply:
Not to my knowledge.